vendor/composer/autoload_static.php

```diff
@@ -271,6 +271,7 @@ class ComposerStaticInitc2822413d75ee1db8fcd50de895c43fa
         'Classic_SEO\\Traits\\Hooker' => __DIR__ . '/../..' . '/includes/traits/class-hooker.php',
         'Classic_SEO\\Traits\\Meta' => __DIR__ . '/../..' . '/includes/traits/class-meta.php',
         'Classic_SEO\\Traits\\Shortcode' => __DIR__ . '/../..' . '/includes/traits/class-shortcode.php',
+        'Classic_SEO\\UpdateClient\\UpdateClient' => __DIR__ . '/../..' . '/includes/class-update-client.php',
```

Comment:
Won't this be overwritten by `composer update`?

Reply:
In this case, no it doesn't. I've not included it as a dependency.